lib/perl/Genome/Model/Tools/Vcf/EvaluateVcf.pm

```diff
@@ -0,0 +1,804 @@
+    if ($tn == 0) {
+        return 0;
+    }
+
+    my $stat =
+      ($results->{true_negatives} - $results->{false_positive_partial}) / $tn;
+    return $stat;
+}
+
+sub stat_exact_ppv {
+    my $self    = shift;
+
+    my $fp_exact = $self->stat_false_positive_exact();
+    my $tp_exact = $self->stat_true_positive_found_exact();
+
+    my $denominator = $fp_exact + $tp_exact;
+    if ($denominator == 0) {
+        return 0;
+    }
+
+    my $stat = $tp_exact / $denominator;
+    return $stat;
+}
+
+sub stat_partial_ppv {
+    my $self    = shift;
+
+    my $fp_partial = $self->stat_false_positive_partial();
+    my $tp_partial = $self->stat_true_positive_found_partial();
+
+    my $denominator = $fp_partial + $tp_partial;
+    if ($denominator == 0) {
+        return 0;
+    }
+
+    my $stat = $tp_partial / $denominator;
+    return $stat;
+}
+
+sub stat_vcf_lines_overlapping_tn {
+    my $self    = shift;
+    my $results = $self->get_rawstats();
+    my $stat    = $results->{false_positives_in_roi};
+    return $stat;
+}
+
+sub stat_lines_specificity_in_tn_only {
+    my $self    = shift;
+    my $results = $self->get_rawstats();
+
+    # this is more accurate than the other specificity measures, but
+    # doesn't take partial into account
+
+    my $tn = $self->stat_true_negatives();
+    if ($tn == 0) {
+        return 0;
+    }
+
+    my $stat = ($tn - $results->{false_positives_in_roi}) / $tn;
+    return $stat;
+}
+
+sub help_brief {
+    return "Compare or Validate VCF files to a Gold Standard";
+}
+
+sub help_detail {
+    my $doc = q{
+
+    BACKGROUND
+    ==========
+
+    Modern genomic pipelines are composed of continually advancing next
+    generation sequencing technologies and software tools.  As pipelines
+    evolve, it is important, especially in a clinical setting, to
+    systematically assess the accuracy and reproducibility of the variant
+    calls that are produced as the end product.
+
+    One way to measure the overall accuracy of a genomic pipeline is to
+    compare variant calls generated from a special input data set with
+    already known variants.  The special data set is referred as a "gold
+    standard", and the process of comparing the pipeline obtained variant
+    calls to the known gold standard variant calls is called "validation".
+
+    The validation process categorizes whether a given variant call
+    produced by a genomic pipeline is correctly identified or missed.  The
+    classifications can be measured overall by the statstical notions of
+    sensitivity, specificity, and positive predictive value (PPV).
+
+    The standard format for storing variant calls is a VCF file.
+    A procedure to validate VCF files against each other is performed in
+    this script.  It roughly follows the NIST recommendations as described
+    in the [Genome in a Bottle paper][1].
+
+    METHODOLOGY
+    ===========
+
+    This tool compares a gold-standard VCF to an experimental VCF and a BED
```

Comment:
Here it is `gold-standard VCF` but in the following paragraph it is `gold standard VCF`.  These should be made consistent.